beacon-chain/sync/rpc_send_request.go

```diff
@@ -198,6 +205,50 @@ func SendBlobSidecarByRoot(
 
 type blobResponseValidation func(blocks.ROBlob) error
 
+func chainBlobValidations(vf ...blobResponseValidation) blobResponseValidation {
+	return func(blob blocks.ROBlob) error {
+		for i := range vf {
+			if err := vf[i](blob); err != nil {
+				return err
+			}
+		}
+		return nil
+	}
+}
+
+type seqBlobValid struct {
+	prev *blocks.ROBlob
+}
+
+func (sbv *seqBlobValid) nextValid(blob blocks.ROBlob) error {
+	if sbv.prev == nil {
+		sbv.prev = &blob
```

Comment:
If there isn't a previous blob, we need to check that the blob index is zero.

Reply:
done